elements/pfe-accordion/demo/pfe-accordion.html

```diff
@@ -1,325 +1,322 @@
+      <pfe-accordion-header>
+        <h3>Laboris sunt qui dolor consectetur excepteur in aliqua ipsum?</h3>
+      </pfe-accordion-header>
+      <pfe-accordion-panel>
+        <p>Culpa adipisicing sunt dolor ullamco dolor duis in ad commodo.</p>
+        <pfe-cta>
+          <a href="#">Call to action</a>
+        </pfe-cta>
+      </pfe-accordion-panel>
+      <pfe-accordion-header>
+        <h3>Anim est tempor fugiat pariatur laborum deserunt ex mollit aliquip?</h3>
+      </pfe-accordion-header>
+      <pfe-accordion-panel>
+        <p><a href="#">Ullamco ullamco sint</a> ex id magna elit deserunt dolore nostrud eu et dolore est Lorem. Esse laborum do ut consectetur occaecat proident et nostrud ut nostrud veniam officia Lorem.</p>
+      </pfe-accordion-panel>
+      <pfe-accordion-header>
+        <h3>Nostrud ad sit commodo nostrud?</h3>
+      </pfe-accordion-header>
+      <pfe-accordion-panel>
+        <p>Nisi veniam tempor reprehenderit laboris amet laborum et do ut. Veniam eiusmod aliquip ullamco quis esse laborum Lorem exercitation consequat.</p>
+      </pfe-accordion-panel>
+      <pfe-accordion-header>
+        <h3>Reprehenderit cupidatat labore?</h3>
+      </pfe-accordion-header>
+      <pfe-accordion-panel>
+        <p>Magna incididunt aliquip consectetur dolor adipisicing amet cillum officia nostrud. Elit exercitation voluptate aute nostrud.</p>
+        <pfe-cta>
+          <a href="#">Call to action</a>
+        </pfe-cta>
+      </pfe-accordion-panel>
+    </pfe-accordion>
+  </pfe-band>
 
-      <pfe-accordion disclosure="false">
-        <pfe-accordion-header>
-          <h3>Not a disclosure because disclosure=false</h3>
-        </pfe-accordion-header>
-        <pfe-accordion-panel>
-          <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut
-            labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
-            et ea rebum.</p>
-          <p>Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor
-            sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore
-            magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet
-            clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-          <p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut
-            labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores
-            et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
-        </pfe-accordion-panel>
-      </pfe-accordion>
-    </article>
+  <pfe-band class="custom-styles" full-width id="custom">
+    <h2>Custom styles</h2>
+    <pfe-accordion id="custom">
```

Comment:
Usage of the same `id` attr  twice, change the pfe-band id to `custom-band` instead.

```Suggestion
  <pfe-band class="custom-styles" full-width id="custom-band">
    <h2>Custom styles</h2>
    <pfe-accordion id="custom">
```